PathOfTerraria.csproj

```diff
@@ -36,6 +36,9 @@
       <HintPath>lib\NPCUtils.dll</HintPath>
     </Reference>
 	</ItemGroup>
+	<ItemGroup>
+	  <Folder Include="Assets\Backgrounds\" />
+	</ItemGroup>
```

Comment:
Probably should remain consistent and remove this, but I don't think it effects anything.